### DISCUSSION TITLE
raspberry buster kodi 18.7 - package version 2.4.8

### DISCUSSION THREAD

**Discussion by Enjoy-Combi** [Catch-up-TV-and-More/plugin.video.catchuptvandmore#631] [General]
Hi,

I was having this issue regarding inputstream.adaptive 2.4.4 provided by raspberry pi os buster
It was difficult to find a newer version, then I have carried out the compilation under raspbrery pi 4 32 bit version.

If this can help someone:
sudo apt remove kodi-inputstream-adaptive
and copy the content of the attached archive to .kodi/addons/ (mv script.module.inputstream.adaptive/ .kodi/addons/)

Then restart and it should be OK.
[script.module.inputstream.adaptive-2.4.8-armhf.zip](https://github.com/Catch-up-TV-and-More/plugin.video.catchuptvandmore/files/8002433/script.module.inputstream.adaptive-2.4.8-armhf.zip)

**Comment by joaopa00** [comment]
Bugs must be reported in ticket in the section Issue.
